src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/ArtworkContextAuction.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { ArtworkContextAuctionFixture } from "Apps/__tests__/Fixtures/Artwork/OtherWorks/ArtworkContexts/ArtworkContextAuction.fixture"
+import { RelayStubProvider } from "DevTools/RelayStubProvider"
+import { mount } from "enzyme"
+import { cloneDeep } from "lodash"
+import React from "react"
+import { ArtworkContextAuctionFragmentContainer } from "../ArtworkContextAuction"
+
+describe("ArtworkContextAuction", () => {
+  const getWrapper = (props = ArtworkContextAuctionFixture) => {
+    return mount(
+      <RelayStubProvider>
+        <ArtworkContextAuctionFragmentContainer
+          artwork={props.artwork as any}
+        />
+      </RelayStubProvider>
+    )
```

Comment:
Personally I think even _just_ these tests are fine....(possibly controversial, ducks a chair thats thrown at me)

Reply:
Theres a wild and steep complexity gradient we've been dealing with lately and I always wonder where the proper balance is /shrug. 

I'm all onboard with Relay at this point but more and more lately I've been wondering what all of this would look like if our queries were plain strings, rather than compiled artifacts, and simple tests backed our architecture rather than complicated infrastructure (simple -> like how we do it in MP). 

All of these Relay handshakes we have to deal with around `@arguments` and so on would vanish since we'd be able to simply interpolate props into our queries and test input / output. 

Assuming test coverage was sufficient, I think we'd achieve a similar level of security as Relay (though with less complexity). We'd also have to figure out TypeScript type generation but there are a lot of resources in the Apollo community.

Reply:
Are.na is a great study in React / GraphQL simplicity: https://github.com/aredotna/ervell/tree/master/react/components